File: Difficulty: Easy/Remove Consecutive Characters/remove-consecutive-characters.py
# User function Template for python3

class Solution:
    def removeConsecutiveCharacter(self, s):
        res = ''
        for i in range(len(s)):
            if i == 0 or s[i] != s[i-1]:
                res = res + s[i]
            else:
                continue
        return res

File: Difficulty: Easy/Remove Consecutive Characters/remove-consecutive-characters.cpp
class Solution {
  public:
    string removeConsecutiveCharacter(string& s) {
        string res = "";
        int n = s.length();
        
        res += s[0];
        
        for (int i = 1; i < n; i++) {
            if (s[i] != s[i - 1]) {
                res += s[i];
            }
        }
        return res;
    }
};